README.md

```diff
@@ -1,57 +1,63 @@
-# QualityMetricsCli
+# Code PushUp CLI
 
-<a alt="Nx logo" href="https://nx.dev" target="_blank" rel="noreferrer"><img src="https://raw.githubusercontent.com/nrwl/nx/master/images/nx-logo.png" width="45"></a>
+**Quality metrics for web projects, highly configurable.**
```

Comment:
How about this:
```suggestion
**Wrap your favourite tool and integrate it into your CI, visualise your results in a beautiful dashboard**
```

Reply:
I'm not sure that answers the question of what it is, though.

How about this? 

> Quality metrics for your web project. Configure what you want to track using your favourite tools, integrate it in your CI and visualize reports in a beautiful dashboard.

Reply:
> Quality metrics for your web project
Maybe just project? as we not only do web stuff? 
Im fine with the current one too. so lets take it.

Reply:
> Maybe just project? as we not only do web stuff?

OK, I'll go with "software project" instead.

> As a potential user that doesn't know about the portal product, won't the dashboard part without any link or reference to the portal be confusing?

Yes, possibly. But we don't have any link to send them to yet, it could be some time before we're ready to market the portal publicly.